patches/tModLoader/Terraria/ID/TileID.TML.cs

```diff
@@ -71,5 +71,20 @@ partial class Sets
 		/// Tiles that are interpreted as a wall by nearby walls during framing, causing them to frame as if merging with this adjacent tile. Prevents wall from drawing within bounds for transparant tiles.
 		/// </summary>
 		public static bool[] WallsMergeWith = Factory.CreateBoolSet(Glass);
+
+		/// <summary>
+		/// New created sets to facilitate vanilla biome block counting including modded blocks. To replace the current hardcoded counts in SceneMetrics.cs
+		/// </summary>
+		public static int[] CorruptBiome = Factory.CreateIntSet(0, 23, 1, 24, 1, 25, 1, 32, 1, 112, 1, 163, 1, 400, 1, 398, 1, 27, -10);
```

Comment:
Either change the summary to a comment, or add a summary to every set (but I doubt the latter is necessary). Currently the `CorruptBiome` sets gets the generic description, and nothing else.